From: Fabian Pfitzner <f.pfitzner@pengutronix.de>
Date: Wed, 7 May 2025 14:00:26 +0200
Subject: [PATCH] undefine _TIME_BITS

zlib does not use time_t, so _TIME_BITS is irrelevant.
This was already fixed in the upstream zlib repo [1].

[1] https://github.com/madler/zlib/commit/a566e156b3fa07b566ddbf6801b517a9dba04fa3

Signed-off-by: Fabian Pfitzner <f.pfitzner@pengutronix.de>
---
 qtwebengine/src/3rdparty/chromium/third_party/zlib/gzguts.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/qtwebengine/src/3rdparty/chromium/third_party/zlib/gzguts.h b/qtwebengine/src/3rdparty/chromium/third_party/zlib/gzguts.h
index 57faf37165a3..d2057e40bcea 100644
--- a/qtwebengine/src/3rdparty/chromium/third_party/zlib/gzguts.h
+++ b/qtwebengine/src/3rdparty/chromium/third_party/zlib/gzguts.h
@@ -7,9 +7,8 @@
 #  ifndef _LARGEFILE_SOURCE
 #    define _LARGEFILE_SOURCE 1
 #  endif
-#  ifdef _FILE_OFFSET_BITS
-#    undef _FILE_OFFSET_BITS
-#  endif
+#  undef _FILE_OFFSET_BITS
+#  undef _TIME_BITS
 #endif
 
 #ifdef HAVE_HIDDEN
